src/Dev/FunctionalTest.php

```diff
@@ -401,15 +405,28 @@ public function assertExactHTMLMatchBySelector($selector, $expectedMatches, $mes
      */
     public function logInAs($member)
     {
-        if (is_object($member)) {
-            $memberID = $member->ID;
-        } elseif (is_numeric($member)) {
-            $memberID = $member;
-        } else {
-            $memberID = $this->idFromFixture('SilverStripe\\Security\\Member', $member);
+        if (is_numeric($member)) {
+            $member = DataObject::get_by_id(Member::class, $member);
+        } elseif (!is_object($member)) {
+            $member = $this->objFromFixture('SilverStripe\\Security\\Member', $member);
         }
 
-        $this->session()->inst_set('loggedInAs', $memberID);
+        $this->logIn($member);
+    }
+
+    /**
+     * Log out the member
+     *
+     */
+    public function logOut()
+    {
+        $this->session()->inst_clear('loggedInAs');
```

Comment:
@tractorcow
Should we make this call logOut() on the current IdentityStore, instead of assuming session?
In fact, for testing, maybe we just use Security::setCurrentUser(), and don't modify session / persistant state?
---
No, we actually need to clear the session for some tests, thus I would prefer to keep it.